src/app/app.component.html

```diff
@@ -1,8 +1,4 @@
-<ng-container *ngIf="graphs == null; else dataAvailable">
-  <img
-    src="https://cdn.dribbble.com/users/1680472/screenshots/5697390/google-loader.gif"
-  />
-</ng-container>
+<ng-container *ngIf="graphs == null; else dataAvailable"> </ng-container>
```

Comment:
(this change request can be left to the next PR)

----

Oo now that the ng-container is empty, let's change this like so:
```diff
-<ng-container *ngIf="condition; else other"> </ng-container>
-<ng-template #other> STUFF </ng-template>
+<ng-container *ngIf="!condition"> STUFF </ng-container>
```

(same as...)

```diff
-if (condition) {}
-else { STUFF }
+if (!condition) { STUFF }
```